hpicker.js

```diff
@@ -262,9 +274,9 @@ class HorizontalPicker extends Component {
     const itemValue = child.props.value;
     const color = this.props.foregroundColor || defaultForegroundColor;
     return (
-      <TouchableWithoutFeedback key={itemValue} onPress={v => this.handleItemPress(v)}>
+      <TouchableWithoutFeedback key={itemValue} onPress={child.props.onPress}>
         <View style={[styles.itemContainer, {width: this.getItemWidth()}]}>
-          <Text style={[styles.itemText, child.props.style, {color}]}>{child.props.label}</Text>
+          <Text style={[styles.itemText, child.props.style, {color}, {opacity: itemValue !== this.props.selectedValue ? 0.4 : 1}]}>{child.props.label}</Text>
```

Comment:
For this I was thinking of adding distinct styles for highlighted items, but never got around taking the time to think proper way. Also I believe this won't animate the change, although, just for opacity it might not matter. Do you have interest taking this a step further?

Reply:
Yes. I've changed  this to become a prop to the main `HorizontalPicker` in a commit just now